docs/admin/install/knative-with-operators.md

```diff
@@ -328,9 +328,14 @@ kubectl logs -f deploy/knative-operator
     kubectl apply --filename {{artifact(repo="serving",file="serving-default-domain.yaml")}}
     ```
 
-    **Caveat**: This will only work if the cluster LoadBalancer service exposes an
-    IPv4 address or hostname, so it will not work with IPv6 clusters or local setups
-    like Minikube. For these, see "Real DNS" or "Temporary DNS".
+    !!! info "Caveat"
```

Comment:
I'd make this a "warning" or something instead since it can potentially cause issues for users, and put it before "We ship..."

Reply:
Would you mind also updating that "we ship" paragraph to something like:

"Knative provides a Kubernetes Job called `default-domain` that configures Knative Serving to use..."

No need for (see caveats) IMO and it's in our style guide now to avoid parentheses afaik.

Reply:
this PR is just correcting the existing caveat, so maybe we could do that in a separate PR?

Reply:
OK, opening an issue